proxy/PoolableSession.h

```diff
@@ -190,3 +204,39 @@ PoolableSession::FQDNLinkage::equal(CryptoHash const &lhs, CryptoHash const &rhs
 {
   return lhs == rhs;
 }
+
+inline void
+PoolableSession::enable_outbound_connection_tracking(OutboundConnTrack::Group *group)
+{
+  ink_assert(nullptr == conn_track_group);
```

Comment:
Not sure I'd do that - passing `nullptr` could be a way to disable connection tracking, and everything should function even if the value is `nullptr`.